test/conformance/config/ptptestconfig.go

```diff
@@ -11,6 +11,11 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const (
+	// Default cpu usage threshold in milliCpus.
+	PtpDefaultMilliCoresUsageThreshold = 15
```

Comment:
In real cluster I saw ptp daemon crossing avg 17 mc . So this could fail  most of the time. Also cpu usage changes for different profile OC/BC and Dual as ptp process increases .